From b5b7a14418a49b40708b8efdc970f477ebcb24b4 Mon Sep 17 00:00:00 2001
From: Snakemonster <nikita.o.platonov@gmail.com>
Date: Wed, 2 Jul 2025 13:02:18 +0300
Subject: [PATCH] Removed unused subdirectories.

---
 Examples/CMakeLists.txt | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/Examples/CMakeLists.txt b/Examples/CMakeLists.txt
index ba5e88f40..d515106a3 100644
--- a/Examples/CMakeLists.txt
+++ b/Examples/CMakeLists.txt
@@ -1,8 +1,9 @@
-add_subdirectory(Algorithms)
-add_subdirectory(Detectors)
+cmake_minimum_required(VERSION 3.14)
+# add_subdirectory(Algorithms)
+# add_subdirectory(Detectors)
 add_subdirectory(Framework)
-add_subdirectory(Io)
-add_subdirectory_if(Python ACTS_BUILD_EXAMPLES_PYTHON_BINDINGS)
-add_subdirectory_if(Scripts ACTS_BUILD_ANALYSIS_APPS)
+# add_subdirectory(Io)
+# add_subdirectory_if(Python ACTS_BUILD_EXAMPLES_PYTHON_BINDINGS)
+# add_subdirectory_if(Scripts ACTS_BUILD_ANALYSIS_APPS)
 
 add_subdirectory(HelloWorld)
-- 
2.49.0

